wrangler/package.json

```diff
@@ -4,7 +4,9 @@
   "module": "./dist/shim.mjs",
   "scripts": {
     "build": "webpack",
-    "deploy": "wrangler publish"
+    "deploy": "wrangler publish",
+    "init": "export $(cat .env|tr -d \"[:blank:]\"|xargs -d '\n') && envsubst < wrangler.toml.dist > wrangler.toml",
```

Comment:
`xargs` works weirdly on mac -> (no -d option exists).

I believe `xargs` defaults to having a new line delimiter on both Linux and mac. I was able to successfully run with this:

```
    "init": "export $(cat .env|tr -d \"[:blank:]\"|xargs) && envsubst < wrangler.toml.dist > wrangler.toml",

```

Reply:
nice catch!